src/app/unlocked-stage/unlocked-stage.component.scss

```diff
@@ -124,43 +137,96 @@
     @include positioning(relative, 50%, 80%);
```

Comment:
add selectable(none) and draggable(none) mixins for the human as well

Reply:
Done.